packages/sui-react-web-vitals/src/index.js

```diff
@@ -113,7 +113,8 @@ export default function WebVitalsReporter({
         path: hasPathOnRoute ? pathname : getNormalizedPathname(pathname),
         target,
         loadState: attribution.loadState,
-        ...(attribution.eventType && {eventType: attribution.eventType})
+        ...(attribution.eventType && {eventType: attribution.eventType}),
+        visibilityState: document.visibilityState
```

Comment:
I'm assuming `ducment` will be available because we're inside a `useMount`

Reply:
Yes, we have access to document inside a `useMount`